org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -30,6 +30,63 @@ Parameters:
     Type: String
     Default: '906769aa66-5d23a723-54f3-4c08-a67b-311e555f4e85'
 
+  adminCentralCfnDeployerGroup:   #JC aws-admincentral-cfn-deployers group
+    Type: String
+    Default: '906769aa66-f7cda126-3384-4fc0-bf6d-c01a00172e6d'
+
+SsoAdminCentralCfnDeployer:
+  Type: update-stacks
+  DependsOn: [ SsoAdminCentralCfnDeployerPolicy ]
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-admincentral-cfn-deployer'
+  StackDescription: 'SSO: deployer role used by admincental CFN deployer group'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref AdminCentralAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref adminCentralCfnDeployerGroup
+    permissionSetName: 'Cloudformation-Deployer'
+    sessionDuration: 'PT12H'
+    inlinePolicy: >-
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [ "s3:ListAllMyBuckets" ],
+            "Resource": "arn:aws:s3:::*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [ "s3:ListBucket", "s3:GetBucketLocation", "s3:getBucketVersioning" ],
+            "Resource": [
+              "arn:aws:s3:::bootstrap-awss3cloudformationbucket-19qromfd235z9",
+              "arn:aws:s3:::essentials-awss3lambdaartifactsbucket-x29ftznj6pqw"
+            ]
+          },
+          {
+            "Effect": "Allow",
+            "Action": [ "s3:PutObject", "s3:PutObjectAcl", "s3:GetObject", "s3:GetObjectAcl", "s3:DeleteObject" ],
+            "Resource": [
+              "arn:aws:s3:::bootstrap-awss3cloudformationbucket-19qromfd235z9/*",
+              "arn:aws:s3:::essentials-awss3lambdaartifactsbucket-x29ftznj6pqw/*"
+            ]
+          },
+          {
+            "Sid": "ManageServerlessRepo",
+            "Effect": "Allow",
+            "Action": [ "serverlessrepo:*" ],
+            "Resource": [
+              "arn:aws:serverlessrepo:us-east-1:531805629419:applications/*"
```

Comment:
This accountid is the organizations account, not admincentral... Your PR comment indicates this should be admincentral.

Reply:
you are correct. fixed. thanks.